helm_deploy/prison-offender-events/templates/housekeeping-cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-queue-housekeeping
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "*/10 * * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 600
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: housekeeping
+              image: quay.io/hmpps/dps-tools
+              args:
+                - /bin/sh
+                - -c
+                - curl -XPUT http://prison-offender-events/queue-admin/retry-all-dlqs
```

Comment:
is that correct url given DNS is offender-events ? (e.g https://offender-events.prison.service.justice.gov.uk/info)

Reply:
iirc the host is the name of the service - we are calling from behind the ingress so hit the service directly

Reply:
This keeps confusing me.  Will check and fix in next PR